PiWeb.Api.Training.Events/Events/CharacteristicCreatedEvent.cs

```diff
@@ -0,0 +1,33 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+using System;
+
+namespace PiWeb.Api.Training.Events.Events;
```

Comment:
Was it intended to change the namespace? How does it work to find the event class when the namespaces are different?

Reply:
Change was intended. As discussed, the events are matched based on method names of the sender hub.